sphinx/random-distributions.rst

```diff
@@ -0,0 +1,47 @@
+Distributions
+=============
+
+Our distribution support will grow over time. If you need a distribution that is not yet supported please `post an issue <https://github.com/mrc-ide/dust/issues/>`_.
```

Comment:
```suggestion
Our distribution support will grow over time. If you need a distribution that is not yet supported please `post an issue <https://github.com/mrc-ide/dust/issues/>`__.
```
I believe is best, otherwise it makes an internal link or something